code/modules/assembly/flash.dm

```diff
@@ -143,6 +143,8 @@
 		return FALSE
 	if(iscarbon(M))
 		flash_carbon(M, user, 5, 1)
+		if HAS_TRAIT(M, TRAIT_FLASHBURN)
+			M.adjustFireLoss (20)

```

Comment:
this feels a bit low; maybe up the damage to 25 or 30? Keep in mind that a lit welder will usually do 15 damage to a nightmare, due to their inability to wear most kinds of armor.